default.nix

```diff
@@ -90,11 +90,9 @@ let haskellPackages =
         # XXX On macOS cabal2nix does not seem to generate a dependency on
         # Cocoa framework.
         buildInputs =
-            [nixpkgs.haskellPackages.inspection-testing] ++
-            (if builtins.currentSystem == "x86_64-darwin"
-             then [nixpkgs.darwin.apple_sdk.frameworks.Cocoa]
-             else []
-            );
+            if builtins.currentSystem == "x86_64-darwin"
```

Comment:
Why remove this?

Reply:
It does not work as expected. We need `inspection-testing` to be installed as a ghc package in the shell which it does not.